gitpod-io/gitpod
# Various prebuild fixes

## Description

This PR fixes a couple of prebuilds-related issues. It originally was set out to fix incomplete endings of logs, but it ended up being about so a couple of other fixes, just not about this one 🤷.

## Related Issue(s)

Fixes ENT-445
Fixes the `NetworkError when attempting to fetch resource` bug

## How to test

1. Try executing some prebuild
2. Optionally, also try to run a failed prebuild (for instance https://github.com/mustard-mh/test/tree/prebuild/image-build-failed) and observe it in real time for ENT-445.

https://ft-fix-suc3e31a702ac.preview.gitpod-dev.com
